### DISCUSSION TITLE
http api is not working

### DISCUSSION THREAD

**Discussion by GauravChoube** [bluenviron/mediamtx#565] [Questions] [answered]
Hi,
I am using rtsp server since 3 month and its working great and helped lot.
New feature of HTTP API , I have tried but seem not working.

As per instruction , i made it yes before api in configuration file.
following are command which i run:
```
 sudo docker run  -it --rm -e RTSP_PROTOCOLS=tcp -p 8554:8554 -p 1935:1935 -p 8888:8888 -p 9997:9997  -v $PWD/rtsp-simple-server.yml:/rtsp-simple-server.yml --name rtspServer aler9/rtsp-simple-server
```

To get the http api, i have used
```
curl http://127.0.0.1:9997/v1/paths/list
```

Please help me and i am not sure where is problem

Regards
Gaurav Choubey

**Comment by aler9** [comment, marked as answer]
For security reasons, the API is exposed on localhost; therefore, you need to change the listen address to

```yml
apiAddress: 0.0.0.0:9997
```

**Reply by GauravChoube** [reply]
Thanks for reply.
Now its working. 
You did really great job

[a comment by an automated account is not shown]
